src/DataValues/QuantityValue.php

```diff
@@ -91,155 +70,59 @@ public function __construct( DecimalValue $amount, $unit, DecimalValue $upperBou
 	 * @note: if the amount or a bound is given as a string, the string must conform
 	 * to the rules defined by @see DecimalValue.
 	 *
-	 * @since 0.1
+	 * @since 0.7
```

Comment:
THis is a sideeffect of renaming this back-and-forth. In the first version of this change, this code was in the new BoundedQuantityValue class. I'll fix all the since tags.